app/lib/backend_api_logic/service_extension.rb

```diff
@@ -9,7 +9,7 @@ module ServiceExtension
       has_many :backend_apis, through: :backend_api_configs
 
       delegate :backend_api, to: :backend_api_proxy
-      delegate :api_backend, to: :backend_api
+      delegate :api_backend, to: :backend_api, allow_nil: true
```

Comment:
Why can it be nil? It must not, that is the whole point of the backend_api_proxy